dev/reference/tools/tidb-lightning/config.md

```diff
@@ -0,0 +1,324 @@
+---
+title: TiDB Lightning Configuration
+summary: Learn about the CLI usage and sample configuration in TiDB Lightning.
+category: reference
+---
+
+# TiDB Lightning Configuration
+
+This document provides samples for global configuration, task configuration, and TiKV Importer configuration in TiDB Lightning, and describes the usage of command-line parameters.
+
+## Configuration files
+
+TiDB Lightning's global and task configurations have compatible structures. Unless [Server mode](/dev/reference/tools/tidb-lightning/web.md) is enabled, the same file is used for both parts of configuration.
+
+### TiDB Lightning (Global)
+
+```toml
+### tidb-lightning global configuration
+
+[lightning]
+# The HTTP port for the web interface and Prometheus metrics pulling (0 to disable).
+status-addr = ':8289'
+
+# Toggle server mode and use of the web interface.
+# See the corresponding section for details.
```

Comment:
it's better to give a link for `the corresponding section`

Reply:
Can't add a link in a code block...

Reply:
I mean let user to copy and past, `which section the corresponding section is` is a problem